app/_src/deck/reference/deck_diff.md

```diff
@@ -65,7 +71,7 @@ This flag can be specified multiple times for multiple files.
 Use `-` to read from stdin. (Default: `[kong.yaml]`)
 
 `-w`, `--workspace`
-:  Diff configuration with a specific workspace (Kong Enterprise only).
+:  Diff configuration with a specific workspace ({{site.ee_product_name}} only).
 This takes precedence over _workspace fields in state files.
```

Comment:
Is `_workspace` a type of field? Or is that a stray underscore?

Reply:
`_workspace` is accurate : https://docs.konghq.com/deck/latest/guides/kong-enterprise/#manage-one-workspace-at-a-time